cypress/integration/app/activateuser.spec.ts

```diff
@@ -0,0 +1,104 @@
+describe("Invite user", () => {
+  it("Enable SMTP setting", () => {
+    cy.setup();
+    cy.login();
+
+    cy.visit("/settings/organization");
+
+    cy.findByLabelText(/enable smtp/i).check({ force: true });
+
+    cy.findByLabelText(/sender address/i)
+      .click()
+      .type("rachel@fleetdm.com");
+
+    cy.findByLabelText(/smtp server/i)
+      .click()
+      .type("localhost");
+
+    cy.get("#port").click().type("{selectall}{backspace}1025");
+
+    cy.findByLabelText(/use ssl\/tls/i).check({ force: true });
+
+    cy.findByLabelText(/smtp username/i)
+      .click()
+      .type("rachelsusername");
+
+    cy.findByLabelText(/smtp password/i)
+      .click()
+      .type("rachelspassword");
+
+    cy.findByRole("button", { name: /update settings/i }).click();
+  });
```

Comment:
I realize now it would be best to set up the email with a command so that we can do it without interacting with the UI (because we've already tested doing this via UI and the test will run quicker that way). We actually have a `setupSMTP` function in the `teams` branch that you can copy over: https://github.com/fleetdm/fleet/blob/teams/cypress/support/commands.ts#L54-L73.

Reply:
Smart. I'll get on this.